client/src/Actions/DepositActions.js

```diff
@@ -31,18 +46,28 @@ export const MakeDeposit = (deposit_details, token) => dispatch => {
     .catch(error => {
       dispatch({ 
         type: types.DEPOSITING_FAILURE, 
-        payload: {error} 
+        payload: {
+          error
+        } 
       });
     })
 }
 
 export const SetDepositAmount = (amount) => dispatch => {
-  return dispatch({
-    type: types.SET_DEPOSIT_AMOUNT,
-    payload: {
-      depositAmount: amount,
-    }
-  })
+  return amount >= 10 ? 
```

Comment:
Is the `return` keyword needed here? I'm not sure what it is doing in this case. Ternaries return values implicitly.

Reply:
After playing around with it for a bit, you're correct. The return statement is superfluous. It executes the same way. 

However, for the sake of readability, I will convert it into a normal conditional statement with explicit returns. It doesn't work as well for people reading the code.